flect_test.go

```diff
@@ -16,8 +16,8 @@ type tt struct {
 func Test_LoadInflections(t *testing.T) {
 	r := require.New(t)
 	m := map[string]string{
-		"beatle": "the beatles",
```

Comment:
This is one example of the bad usage of Flect that support parts. "the beatles" will be parsed as "the" + "beatles" and singularize and pluralize function will only see the last part, "beatles", and the result of singularization for "the beatles" will always have "the" part.